sdk/communication/Azure.Communication.AlphaIds/src/Azure.Communication.AlphaIds.csproj

```diff
@@ -17,6 +17,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
+	<PackageReference Include="System.ClientModel" />
```

Comment:
I don't think these are needed with this https://github.com/Azure/azure-sdk-for-net/blob/main/eng/Directory.Build.Common.targets#L257